src/App.js

```diff
@@ -53,13 +56,15 @@ class Root extends React.Component {
     WebSocketHandler.on('dashboard', this.handleWebsocket);
 
     hathorLib.axios.registerNewCreateRequestInstance(createRequestInstance);
+    this.props.apiLoadErrorUpdate({apiLoadError: false});
 
     versionApi.getVersion().then((data) => {
       hathorLib.network.setNetwork(data.network.split('-')[0]);
       this.props.isVersionAllowedUpdate({allowed: helpers.isVersionAllowed(data.version)});
     }, (e) => {
       // Error in request
       console.log(e);
+      this.props.apiLoadErrorUpdate({apiLoadError: true});
```

Comment:
I like this but I understand that this only works if the first request fails (the version request), right? This is the home screen and it already help us in the case the screen becomes white when the explorer API is not working. I think we should keep this PR only with this but create an issue to add this `apiLoadError: true` redux value when other APIs fail as well.

Reply:
Sure. I agree. I just think we have to discuss how we'll show it on page, because it can be a case of service shutdown, a momentary problem, just that specific endpoint not working or a 402 error, for example.

Reply:
I agree that we should work on it in a separate PR. So, we should at least create a new issue for that.

Reply:
Issue created: https://github.com/HathorNetwork/hathor-explorer/issues/57